models/src/test/java/io/sphere/sdk/reviews/commands/ReviewsEndpointTest.java

```diff
@@ -0,0 +1,20 @@
+package io.sphere.sdk.reviews.commands;
+
+import io.sphere.sdk.reviews.Review;
+import io.sphere.sdk.reviews.queries.*;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+/**
+ * Created by njacinto on 21/04/15.
+ */
+public class ReviewsEndpointTest {
```

Comment:
This is implicitly tested by the integration tests. So you can delete this class.